tasks/delete_old_reconcile_jobs/test/unit_tests/test_delete_old_reconcile_jobs.py

```diff
@@ -171,7 +170,6 @@ def test_delete_jobs_older_than_x_days_error_logged_and_raised(
     def test_handler_happy_path(
         self,
         mock_task: MagicMock,
-        mock_LOGGER: MagicMock,
```

Comment:
No luck in adding the `inject_lambda_context` decorator to tests, eh? Oh well.

Reply:
not presently. Added instructions to look further into this in the existing card ORCA-569